docs/source/recipes/images.md

```diff
@@ -0,0 +1,64 @@
+---
+myst:
+  html_meta:
+    "description": "Volto code recipe for images"
+    "property=og:description": "Volto code recipe for images"
+    "property=og:title": "Volto code recipe for images"
+    "keywords": "Plone, Volto, code, recipe, images"
+---
+
+(images-label)=
+
+# Images
+
+Volto comes with an `Image` component that loads images in an optimal way.
+
+Features of the `Image` component are the following.
+
+- optimized [lazy loading](https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/loading)
+- automatic `srcset` generation, if given a content item or catalog brain
+- optional responsive width
+
+The component can accept all the regular HTML `<img>` attributes and a few extra for the above features.
+
+Setting the `loading="lazy"` attribute to the `Image` component also adds the `decoding="async"` attribute.
+The [`async`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/decoding) attribute decodes the image asynchronously to reduce delay in presenting other content.
+
+To render an image from a content object, you can use the component as shown in the following code snippet.
+
+```jsx
+import { Image } from '@plone/volto/components';
+
+<Image item={content} imageField="image" alt="" />;
+```
+
+This will generate the `src`, `srcset`, `width`, and `height` attributes automatically for the given image.
+
+If the `responsive` prop is added, a class is applied that will make the image width responsive.
+
+## Serving the right image size
+
+To serve the proper image size for the viewer's device, you should add a `sizes` attribute to the `Image` component while rendering.
+This responsibility is left to the developer because it depends on where the image is rendered.
+
+For example, if the image is rendered in a container that is always half as wide as the page, the `sizes` attribute should reflect that to properly inform the browser which image size to request and render.
+
+```jsx
+<Image item={content} imageField="image" alt="" sizes="50vw" />
+```
+
+If the image is rendered at full viewport width when the viewport is less than `900px` wide, though, you can add a media condition to the `sizes` attribute.
+
+```jsx
+<Image
+  item={content}
+  imageField="image"
+  alt=""
+  sizes="(max-width: 900px) 100vw, 50vw"
+/>
+```
+
+More information on this subject can be found at the following resources.
+
+- https://developer.mozilla.org/en-US/docs/Learn/HTML/Multimedia_and_embedding/Responsive_images
+- https://www.builder.io/blog/fast-images
```

Comment:
@stevepiercy  Is it mandatory to avoid using angle brackets for bare links? I have markdownlint installed in vscode and it really complains about removing the angle brackets :D I can deal with it if needed, of course, I was just wondering

Reply:
It is an optional style guide. Please do whatever makes your job easier or scratches your itch.

However, I would suggest the [MyST-Markdown VS Code extension](https://marketplace.visualstudio.com/items?itemName=ExecutableBookProject.myst-highlight) for better parsing of MyST. MyST syntax is extended CommonMark markdown syntax. We also use several MyST and Sphinx extensions which might get flagged by code editors.

Reply:
I did not know it existed. I will gladly try switching extensions and see how it goes :)